orchestration/templates/ingest-raw-archive.yaml

```diff
@@ -1,239 +0,0 @@
-apiVersion: argoproj.io/v1alpha1
```

Comment:
NOTE: The logic here was already fully replicated in a previous PR, we just forgot to delete this file.